packages/json-hash/test/index.test.js

```diff
@@ -0,0 +1,173 @@
+import { cloneDeep } from "lodash";
+import { computeHash } from "../src";
+import { getNodeJSAlgoName, isAvailableAlgo } from "../src/algorthms";
+
+describe("json-hash-node-js", () => {
+  const objectToHash = { name: "Siri", age: 30, country: "Sri Lanka" };
+
+  beforeAll(() => {
+    global.unit_tests_running = true;
```

Comment:
I think we won't need this, this flag was used to handle a few edge cases of tests in a couple of other packages

Reply:
Will remove

Reply:
Done